frontend/src/components/SelectFilterControls/SelectFilterControls.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from "react";
+import classNames from "classnames";
+import { Button } from "solo-uswds";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import classes from "./SelectFilterControls.module.scss";
+
+interface SelectFilterControls {
+  onSubmit: (filter: { option: string; value: string }) => void;
+  options: Array<{ name: string; value: string }>;
+  defaultOption?: string;
+}
+
+const SelectFilterControls: React.FC<SelectFilterControls> = ({
+  options,
+  defaultOption,
+  onSubmit
+}) => {
+  const [currentOption, setCurrentOption] = useState(defaultOption ?? "");
+  const [currentValue, setCurrentValue] = useState("");
+
+  const onOptionChange: React.ChangeEventHandler<HTMLSelectElement> = event => {
+    setCurrentOption(event.currentTarget.value);
+  };
+
+  const onValueChange: React.ChangeEventHandler<HTMLInputElement> = event => {
+    setCurrentValue(event.currentTarget.value);
+  };
+
+  useEffect(() => {
+    setCurrentValue("");
+  }, [currentOption]);
```

Comment:
Do we think we should reset the input? 🤷‍♂ not sold. Would love to know your thoughts.

Reply:
Not sure, I guess if filling out the input and then switching the select you might not expect it to be cleared?

Reply:
agreed and updated per our conversation